gallery/wdio.conf.js

```diff
@@ -69,7 +72,7 @@ exports.config = {
     // Define all options that are relevant for the WebdriverIO instance here
     //
     // Level of logging verbosity: trace | debug | info | warn | error | silent
-    logLevel: 'info',
+    logLevel: 'warn',
```

Comment:
Did you mean to update this?

Reply:
> Did you mean to update this?

Yes.  It spits out what I consider to be debug-level details (full HTTP headers and bodies) of every single call to the selenium server at the INFO level.  It's a large amount of output that is in no way useful to me.